### PR TITLE
Fixed Navbar and made some changes

### PR DESCRIPTION
**Pull request by Gahlotkumkum**
# Pull Request Format
### Issue #73
###Close #130
## Type of PR
- **Add X in the box to specify the improvement type.**
- [X] Bug fix
- [x] Feature enhancement
- [ ] Documentation update
- [x] Other (specify): ___________  

## Description
This pull request aims to [briefly describe what the pull request does, e.g., "add a new feature," "fix a bug," "improve documentation," etc.]. 

## Screenshots / Videos (if applicable)
**Before:**
- [Describe the state before the changes, e.g., "The section headings in the footer were misaligned, leading to an inconsistent and cluttered appearance."]

**After:**
- [Describe the state after the changes, e.g., "The section headings in the footer are now properly aligned, providing a clean and professional look."]

## Checklist
- **Add X in the box to specify.**
- [X] I have performed a self-review of my code.
- [ ] I have tested the changes thoroughly before submitting this pull request.
- [ ] I have provided relevant issue numbers, screenshots, and videos after making the changes.
- [ ] I have commented my code, particularly in hard-to-understand areas.

### Additional Context
[Provide any additional context about the changes, such as specific challenges you faced or decisions you made. This can help reviewers understand the rationale behind your changes.]

Thank you for reviewing my pull request!

### CODE REVIEW

**Comment by Gahlotkumkum** [comment]
@swaraj-das Check this and merge it.

File: index.html [hunk 1]
```diff
@@ -1,307 +1,353 @@
...
+        <a href="/"
+          ><span class="website-name">Collect your GamingTools</span></a
+        >
+        <nav>
+          <ul id="menuList">
+            <li><a href="#">Game Controllers</a></li>
+            <li><a href="#">VR Accessories</a></li>
+            <li><a href="#">Media Remotes</a></li>
+            <li><a href="#">Others</a></li>
+            <li class="login"><a href="login/login.html">Login</a></li>
```
Comment (swaraj-das) [inline index.html:24]: Change the login button list element to button...and add hover effect.

**Comment by swaraj-das** [comment]
Don't add hover effect to the navbar and fix the navbar

**Comment by Gahlotkumkum** [comment]
Okay on it.

**Comment by Gahlotkumkum** [comment]
First, merge it, then I'll clone it again to start working.

**Comment by Gahlotkumkum** [comment]
Hey @swaraj-das consider it to merge in the main branch!!

**Comment by swaraj-das** [comment]
@Gahlotkumkum Merge conflicts. Please resolve it.

**Comment by Gahlotkumkum** [comment]
@swaraj-das  Check this

**Pull request closed without merging**